### PR TITLE
Update DescriptorInterface.php

### PR DESCRIPTION
**Pull request by shreypuranik**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | no
| New feature?  |no
| BC breaks?    | no     
| Deprecations? | no

This update adds a more verbose docblock to the describe method defined in the DescriptorInterface

### CODE REVIEW

File: src/Symfony/Component/Console/Descriptor/DescriptorInterface.php [hunk 1]
```diff
@@ -21,7 +21,8 @@
 interface DescriptorInterface
 {
     /**
-     * Describes an object if supported.
+     * Describes an object if the class that
+     * the object is an instance of, is supported.
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Console/Descriptor/DescriptorInterface.php]: the comma should be removed and the description put on a single line to me.

Comment (shreypuranik) [reply src/Symfony/Component/Console/Descriptor/DescriptorInterface.php]: Hi @nicolas-grekas, thank you for reviewing my Pull Request. I have pushed up a further commit for this update. Please can you review whenever you have a chance to do so?

**Review by nicolas-grekas — APPROVED** [review]

File: src/Symfony/Component/Console/Descriptor/DescriptorInterface.php [hunk 2]
```diff
@@ -21,7 +21,7 @@
 interface DescriptorInterface
 {
     /**
-     * Describes an object if supported.
+     * Describes an object if the class that the object is an instance of is supported.
```
Comment (javiereguiluz) [inline src/Symfony/Component/Console/Descriptor/DescriptorInterface.php]: If we're improving this description, I'd love to improve it even further to explain things better.

1) *"Describes an object ..."*  What does "describe" an object mean? Does it return the main PHPdoc description of the class? Does it return an array of the names of the properties and methods of the object?

2) *"... if the object is supported."*  By who or how? This interface doesn't have a `supports()` method, so I don't understand how can I support or not support objects here.

Thanks!

**Comment by nicolas-grekas** [comment]
Rethinking about it and taking Javier's comment into account, I'm closing because there is no clear description of it in a few words. Also because this interface is not really commonly implemented in userland. Thanks for proposing.

**Pull request closed without merging**